zookeeper-server/src/main/java/org/apache/zookeeper/CreateMode.java

```diff
@@ -102,7 +105,7 @@ public boolean isTTL() {
         return isTTL;
     }
 
-    public int toFlag() {
+    public int getFlag() {
```

Comment:
Even though getFlag() is more conventional, I think toFlag() makes more sense here